server/api/users.js

```diff
@@ -53,3 +53,68 @@ router.get('/:id/latestguesses', async (req, res, next) => {
 		next(error)
 	}
 })
+
+const getRandomInt = max => {
+	return Math.floor(Math.random() * max)
+}
+
+const completedPrevGameOrStartNewGame = async (req, res, next) => {
+	try {
+		let latestWordle = await WordleGame.findAll({
+			where: { userId: req.params.id },
+			include: AcceptedGuess
+		})
+		latestWordle ? (latestWordle = latestWordle[latestWordle.length - 1]) : null
+		if (!latestWordle || latestWordle.acceptedGuesses.length === 5) {
+			next()
+		} else {
+			console.log('not finished game yet')
+		}
+	} catch (error) {
+		next(error)
+	}
+}
+
+//create a new target word and asign that to a new game - send back the new target word to client
+router.get(
+	'/:id/newTargetWord',
+
+	completedPrevGameOrStartNewGame,
+	async (req, res, next) => {
+		try {
+			const randomWord = await AcceptedWord.findByPk(getRandomInt(10656))
```

Comment:
Target words and accepted words should be different tables, and not connected! Accepted words are words that users are allowed to guess, but not ever word a user guesses should be the target word of the day.

Reply:
To clarify about this, target words are words that are relatively easy to guess that everyone knows. So the target word list is relatively short compared to accepted words list. Accepted word lists is basically all 5 letter words that someone could possibly think of and guess!